scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.js

```diff
@@ -89,6 +89,11 @@ export function ContentProfileSchemaEditor(gettext, content) {
                 'underline', 'italic', 'bold', 'table'
             ];
 
+            scope.formatingOptionsEditor3 = [
+                'h1', 'h2', 'orderedlist', 'quote', 'image', 'anchor',
```

Comment:
We have `unorderedlist` too. Also `h3`...`h6` (not sure if needed, but maybe at least `h3`?)